plugins/query-insights/src/main/java/org/opensearch/plugin/insights/rules/model/MetricType.java

```diff
@@ -93,10 +93,9 @@ public static Set<MetricType> allMetricTypes() {
     public int compare(final Number a, final Number b) {
         switch (this) {
             case LATENCY:
-                return Long.compare(a.longValue(), b.longValue());
-            case JVM:
             case CPU:
-                return Double.compare(a.doubleValue(), b.doubleValue());
+            case MEMORY:
+                return Long.compare(a.longValue(), b.longValue());
```

Comment:
@kaushalmahi12 - While I like MEMORY better, I am wondering if similar change is feasible for SearchBackpressure / Query Sandboxing. If not, should we have JVM at both the places?

Reply:
We had some similar discussions before and I remember @msfroh mentioned we should be avoiding the term "JVM" and use `Memory` instead. :P

Reply:
I also like Memory over JVM, since it is language/platform agnostic, but want to ensure consistency across features within Opensearch. @kaushalmahi12 - Thoughts?